maven/api-maven-archive/src/main/java/org/jboss/shrinkwrap/resolver/api/maven/archive/assembler/ArchiveMavenAssembler.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * JBoss, Home of Professional Open Source
+ * Copyright 2012, Red Hat Middleware LLC, and individual contributors
```

Comment:
We actually shouldn't be putting copyright notices on source files anymore. :)  Recommend updating IDE templates.